src/esp/physics/PhysicsManager.h

```diff
@@ -0,0 +1,211 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
+#pragma once
+
+#include <map>
+#include <memory>
+#include <string>
+#include <vector>
+
+/* Bullet Physics Integration */
+#include <Magnum/Trade/MeshData3D.h>
+
+#include "RigidObject.h"
+#include "esp/assets/Asset.h"
+#include "esp/assets/BaseMesh.h"
+#include "esp/assets/CollisionMeshData.h"
+#include "esp/assets/GenericInstanceMeshData.h"
+#include "esp/assets/MeshData.h"
+#include "esp/assets/MeshMetaData.h"
+#include "esp/scene/SceneNode.h"
+
+namespace esp {
+
+namespace assets {
+class ResourceManager;
+}
+
+namespace physics {
+
+class PhysicsManager {
+ public:
+  // explicit PhysicsManager(assets::ResourceManager& _resourceManager) :
+  // resourceManager(_resourceManager) {};
+  explicit PhysicsManager(assets::ResourceManager* _resourceManager) {
+    resourceManager_ = _resourceManager;
+  };
+
+  virtual ~PhysicsManager();
+
+  //============ Initialization =============
+  // load physical properties and setup the world
+  // do_profile indicates timing for FPS
+  virtual bool initPhysics(
+      scene::SceneNode* node,
+      assets::PhysicsManagerAttributes physicsManagerAttributes);
```

Comment:
```suggestion
      const assets::PhysicsManagerAttributes& physicsManagerAttributes);
```
I believe this can be a const reference.